python_modules/dagster-graphql/dagster_graphql_tests/graphql/test_scheduler.py

```diff
@@ -437,6 +603,23 @@ def test_next_tick(graphql_context):
         assert len(tick["evaluationResult"]["runRequests"]) == 1
 
 
+def test_ticks_from_timestamp(graphql_context):
+    schedule_selector = infer_schedule_selector(graphql_context, "past_tick_schedule")
+
+    # get schedule past ticks
+    cur_timestamp = get_timestamp_from_utc_datetime(get_current_datetime_in_utc())
+    result = execute_dagster_graphql(
+        graphql_context,
+        GET_SCHEDULE_QUERY,
+        variables={"scheduleSelector": schedule_selector, "ticksAfter": cur_timestamp},
+    )
+
+    ticks = result.data["scheduleOrError"]["ticksFromTimestamp"]
+    assert len(ticks) == 6
+    assert len([tick for tick in ticks if tick > cur_timestamp]) == 3
+    assert len([tick for tick in ticks if tick <= cur_timestamp]) == 3
+
```

Comment:
add testing to cover the implementation complexity you have comments for like preventing ticks in forward and backward iterators on exact match

Reply:
this test does cover the forward and backward iterator case - will add a comment denoting that though